iml-agent/src/action_plugins/ldev.rs

```diff
@@ -237,4 +241,50 @@ mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_parsing_commented_data() -> Result<(), ImlAgentError> {
```

Comment:
I feel like parsing the file gets too much attention for a plugin which is meant to overwrite the file :)